Ball of Duty Server/Domain/Maps/Map.cs

```diff
@@ -227,6 +230,8 @@ public void Update(Observable observable, object data)
                 {
                     Character killer = (Character)go;
                     killer.AddKill(victim);
+                    Broker.KillNotification(victim.Id, killer.Id);
```

Comment:
this entire method can be called by bullet when it collides with a character.
map subscribes to character which will notify map when it gets a kill